### ISSUE TITLE
"Trying to access array offset on value of type bool" when running "bee help site-install"

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/b#161]
As the issue title suggests, when running `bee help site-install` or `bee help si`, I get this error shown among the help output:

```
Trying to access array offset on value of type bool
 /usr/local/bin/bee/includes/render.inc:355
```

**Comment by ghost** [comment]
Hmm, the code in question seems to be this line: `if ($index == $row['index']) {`

And my understanding of the error message is that `$row` is a boolean. While `$row` can be set to `FALSE` elsewhere in the code, this function isn't run unless it equates to `TRUE` (which is only when it's an array with values)...

@klonos Can you reproduce this using the `.lando` file that comes with Bee?

**Comment by yorkshire-pudding** [comment]
I can reproduce using the bee lando recipe

**Comment by yorkshire-pudding** [comment]
The error disappears if the following example line is commented out:
```php
'bee install --db-name=backdrop --db-user=admin --db-password=P@ssw0rd! --username=Root --password=N0tS3cur3 --email=root@mydomain.com --site-name="My awesome site!"' => bt('Install Backdrop using the given options, and be prompted for the rest.'),
```

Why is it splitting at `--si` (138 characters) and putting `te-name="My awesome site!"` after the description and after the error message?

```
 bee install --db-name=backdrop --db-user=admin --db-password=P@ssw0rd! --username=Root --password=N0tS3cur3 --email=root@mydomain.com --si   Install Backdrop using the given options, and be prompted for the rest.

Trying to access array offset on value of type bool
 /app/includes/render.inc:355

 te-name="My awesome site!"
```
Something to do with the column width calcs in `bee_get_remaining_rows`?

**Comment by yorkshire-pudding** [comment]
My thought at the moment is that we should change the way examples are presented so they are not in columns but example on one line and explanation on the line below.

**Comment by indigoxela** [comment]
It may be that this is a wider problem (first column width seems to have some limitation), but as a "cheap trick" we could make it narrower by adding newlines.

A PR is available for testing / discussion #233.

**Comment by yorkshire-pudding** [comment]
Interesting idea, and while it gets rid of the bug issue, there is still the messiness of the explanation wrapping to column 1.
I'd be open to going with this to get the error cleared with perhaps another ticket for going to separate lines for a cleaner look.
@klonos - do you have any thoughts?
